<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaVideo</class>
 <widget class="QDialog" name="VentanaVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>segundos</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="boton_examinar">
     <property name="text">
      <string>Examinar</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="boton_generar_video">
     <property name="text">
      <string>Generar vídeo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="boton_seleccionar_ejecucion">
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ruta de almacenamiento</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QSpinBox" name="spin_duracion">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>3600</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ejecución a representar </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="edit_ruta"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Duración del vídeo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="edit_ejecucion">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
